chatting/functional_tests/tests.py

```diff
@@ -46,7 +74,7 @@ def check_basic_layout(self):
 
     def test_new_visitor(self):
         # execute browser
-        self.browser.get(self.live_server_url)
+        self.browser.get('http://localhost:8000/accounts/profile/')
```

Comment:
I think so.
it's kind of fixture loading problem.
follow @bbayoung7849 's way. test first :)